package.json

```diff
@@ -74,7 +74,8 @@
   "devDependencies": {
     "ava": "^0.19.1",
     "jest": "^20.0.4",
-    "nyc": "^10.0.0"
+    "nyc": "^10.0.0",
+    "rss": "^1.2.2"
```

Comment:
This would actually not be a dev dependency, just a regular dependency. Dev dependencies are typically only installed when running `npm install` from the root of this project, but since electric is a dependency of `electric-cli`, it's dev dependencies won't be installed when running `npm i -g electric-cli`.